libs/zsdl/src/sdl2.zig

```diff
@@ -1251,6 +1251,10 @@ pub fn pollEvent(event: ?*Event) bool {
 }
 extern fn SDL_PollEvent(event: ?*Event) i32;
 
+/// pub fn pushEvent(event: ?*Event) i32 {
+pub const pushEvent = SDL_PushEvent;
+extern fn SDL_PushEvent(event: ?*Event) i32;
```

Comment:
* the `event` arg should be of type `*Event`, as it's not valid to pass null to this API
* Because `SDL_PushEvent` only returns a result code we should check for this and change the return type of `pushEvent` to `Error!void`. See `showSimpleMessageBox` for an example of this.

Reply:
I think we should use `Error!bool` (`true` == added to the queue, `false` == filtered). What do you think?

Reply:
Another option is `Error!bool`, true for event added and false for event filtered.

Reply:
@michal-z That sounds good to  me.

Reply:
Good catch. Sounds good to me too 👍